sha3/sha3.go

```diff
@@ -0,0 +1,27 @@
+package sha3
```

Comment:
I would prefer to have the `sha3` package under a packaged named `xcrypto`, so it is clear that we are providing a replacement for `x/crypto` and not for the standard library.